app/(auth)/AuthSessionStatus.tsx

```diff
@@ -1,4 +1,11 @@
-const AuthSessionStatus = ({ status, className, ...props }) => (
+const AuthSessionStatus = ({
+    status,
+    className,
+    ...props
+}: {
+    status: string | null
+    className?: string
```

Comment:
you can remove this, because `React.HTMLAttributes<HTMLDivElement>` already includes `className`

Reply:
done, thanks